recipes/foamalgo/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "foamalgo" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "cda69b38c75c867d8037e44b94a238b7d1fd9d7d9bc46760dd32f4128e321a25" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zhujun98/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True
```

you are skipping all the builds here
please remove this line, or if you want it just for linux, please skip just windows platforms
```suggestion
  skip: True  # [win]
```

---

@marcelotrevisani Thanks a lot for the reviews! Indeed, I wanted to know how to skip the windows build.